wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -151,7 +148,7 @@ public Builder() {
      * Returns a byte string with this message's unknown fields. Returns an empty byte string if
      * this message has no unknown fields.
      */
-    public ByteString buildUnknownFields() {
+    public final ByteString buildUnknownFields() {
```

Comment:
These changes aren't strictly needed, but they feel correct since the superclass is the only one which manages the unknown field data.